tests/test_json.py

```diff
@@ -157,16 +167,16 @@ def test_mappable_register():
     assert not new_mapped_seq.is_register_mappable()
 
 
-def test_rare_cases(patch_plt_show):
+def test_rare_cases(patch_plt_show, helpers):
     reg = Register.square(4)
     seq = Sequence(reg, DigitalAnalogDevice)
     var = seq.declare_variable("var")
 
     wf = BlackmanWaveform(var * 100 // 10, var)
     with pytest.warns(
         UserWarning, match="Calls to methods of parametrized objects"
-    ), pytest.raises(
-        ValueError, match="Serialization of calls to parametrized objects"
+    ), helpers.raises_all(
+        [ValueError], match="Serialization of calls to parametrized objects"
```

Comment:
Why are you using the `helpers` in this case?

Reply:
Oops, removing.